test/indexer/test_hca_indexer.py

```diff
@@ -1335,6 +1335,76 @@ def test_multiple_samples(self):
+        self._index_canned_bundle(bundle_fqid)
+        # Sample (Organoid): df23c109, Donor: none
+        bundle_fqid = self.bundle_fqid(uuid='0722b70c-6778-423d-8fe9-869e2a515d35',
+                                       version='2021-03-18T11:38:49.863Z')
+        self._index_canned_bundle(bundle_fqid)
+        donor = {
+            'document_id': 'b111e5bf-e907-47f9-8eed-75b2ec5536c5',
+            'biomaterial_id': 'Human_62',
+            'biological_sex': 'male',
+            'genus_species': ['Homo sapiens'],
+            'development_stage': 'human adult stage',
+            'diseases': ['normal'],
+            'organism_age': '62 year',
+            'organism_age_value': '62',
+            'organism_age_unit': 'year',
+            'organism_age_range': {
+                'gte': 1955232000.0,
+                'lte': 1955232000.0
+            }
+        }
+        donor_none = {
+            k: [None] if isinstance(v, list) else None
+            for k, v in donor.items()
+            if k != 'organism_age_range'
+        }
+        aggregate_donor = {
+            'donor_count': 1,
+            'donor_count_': 1,
+            **{
+                # The `organism_age_range` field will not have a `None`
+                # value since the field is only added to the donor
+                # inner entity if it has `organism_age_in_seconds` value.
+                # FIXME: The donor inner entity of a project aggregate that
+                #        includes a project without a donor should include
+                #        `None` in each of the aggregated donor fields.
+                #        https://github.com/databiosphere/azul/issues/3152
+                k: (v if isinstance(v, list) else [v]) +
+                   ([] if k == 'organism_age_range' or True else [None])
+                for k, v in donor.items()
+            }
+        }
+        hits = self._get_all_hits()
+        for hit in hits:
+            contents = hit['_source']['contents']
+            entity_type, aggregate = self._parse_index_name(hit)
+            if entity_type == 'projects':
+                if aggregate:
+                    self.assertElasticsearchResultsEqual([aggregate_donor], contents['donors'])
+                else:
+                    sample_id = one(contents['samples'])['document_id']
+                    if sample_id == '70d2b85a-8055-4027-a0d9-29452a49d668':
+                        self.assertEqual([donor], contents['donors'])
+                    elif sample_id == 'df23c109-59f0-46d3-bd09-660175b51bda':
+                        # FIXME: The donor inner entity for a project without a donor
+                        #        should have all the standard fields of a donor inner
+                        #        entity with values of `None`.
+                        #        https://github.com/databiosphere/azul/issues/3152
+                        self.assertEqual([donor_none] if False else [], contents['donors'])
```

Comment:
```suggestion
                        self.assertEqual([] if True else [donor_none], contents['donors'])
```

To match the "disabler" for the other FIXME.